integration-tests/hibernate-search-elasticsearch/src/test/resources/elasticsearch-maven-plugin/configuration/elasticsearch.yml

```diff
@@ -10,6 +10,32 @@
 # Please consult the documentation for further information on configuration options:
 # https://www.elastic.co/guide/en/elasticsearch/reference/index.html
 #
+# ---------------------------------- Cluster -----------------------------------
```

Comment:
suggestion: cluster formation in ES is very slow, by design. The configuration for integration tests I had created for Hibernate Search disables all forms of clustering, making it much lighter to boot. Might want to copy that one?

Reply:
P.S. I guess what I had done no longer applies on the latest ES versions, so ignore me if we no longer have that.

Reply:
IIUC this file was copied from Search itself.

Your optimizations came down to disabling multicast, which was removed in ES5: https://github.com/elastic/elasticsearch/issues/22909#issuecomment-276726390

So this may not be optimized as much as what you did, but previous optimizations can no longer be applied, so we'd have to find new ones.

Right now integration tests in Search are reasonably quick, so optimizing this isn't a priority.